### PR TITLE
Delete Intel_Fellowships.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#10595

### CODE REVIEW

File: src/chrome/content/rules/Intel_Fellowships.com.xml [hunk 1]
```diff
@@ -1,18 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://intelfellowships.com/ => https://intelfellowships.com/: (6, 'Could not resolve host: intelfellowships.com')
-Fetch error: http://www.intelfellowships.com/ => https://www.intelfellowships.com/: (6, 'Could not resolve host: www.intelfellowships.com')
-
-	For other Intel coverage, see Intel.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/Intel_Fellowships.com.xml]: Please also delete the obsolete reference in Intel.xml

Comment (koops76) [inline src/chrome/content/rules/Intel_Fellowships.com.xml]: @J0WI I had to do it in a separate PR since I have deleted the repository: #10959

Comment (J0WI) [inline src/chrome/content/rules/Intel_Fellowships.com.xml]: Could you therefore merge this into #10959?

Comment (koops76) [inline src/chrome/content/rules/Intel_Fellowships.com.xml]: @J0WI Done.
